src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -598,6 +598,12 @@ internal static void GetBytes(in decimal d, byte[] buffer)
             buffer[15] = (byte)(d.flags >> 24);
         }
 
+        internal static void GetBytes(in decimal d, byte[] buffer)
```

Comment:
Despite being internal, I did not remove it out of fear of breaking anything. We can mark it as obsolete or entirely remove it though, if nothing else breaks.